include/system-info/class-bd-system-info-page.php

```diff
@@ -189,6 +189,12 @@ protected function render_body() {
 
 <?php bd_print_current_plugins(); ?>
 
+<?php if ( ! empty( _get_cron_array() ) ){ ?>
```

Comment:
Only in PHP 5.5 and above you can pass values that are not variables to the `empty` function.

This line will not work in PHP 5.3 and 5.4. We need to support PHP 5.3 as well.

You can see this issue because the Travis CI failed.